packages/resources/src/Construct.ts

```diff
@@ -21,7 +21,7 @@ export interface SSTConstruct {
 
 export function getFunctionRef(fn?: any) {
   if (!fn) return undefined;
-  if (!(fn instanceof Fn)) return undefined;
+  if (!(fn instanceof lambda.Function)) return undefined;
```

Comment:
Let's check for `sst.Function` here, so the `getFunctionRef` returns undefined for `lambda.Function`. This way, we can probably remove all the handling on the frontend.

Reply:
done!

Reply:
I changed it but I still need the frontend handling.